extensions/amp-story/1.0/amp-story.js

```diff
@@ -2583,6 +2582,27 @@ export class AmpStory extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Shows the play/pause icon if there is a playable element in the story
+   * @private
+   */
+  updatePausedIcon_() {
+    const containsPlayableElement =
+      scopedQuerySelectorAll(
```

Comment:
Nit: `!!scopedQuerySelector()` so we only get the first result, which might be a tiny bit faster